python/sdk/test/integration_test.py

```diff
@@ -1115,6 +1123,13 @@ def test_redeploy_model(integration_test_url, project_name, use_google_oauth, re
 
     # Check that the endpoint remains the same
     assert endpoint.url == new_endpoint.url
+
+    # Check that the deployment configs of v2 have remained the same as those used in v1
+    assert endpoint.resource_request.cpu_request == "123m"
+    assert endpoint.resource_request.memory_request == "234Mi"
+    assert endpoint.env_vars == {"green": "TRUE"}
+    assert endpoint.deployment_mode == DeploymentMode.RAW_DEPLOYMENT
```

Comment:
Additional checks to ensure the original configs do not get overwritten by the default configs.

Reply:
Ahh, that's actually not how the API server works right now 👀 It seems like the intended behaviour (from the implementation in the API server) for the update endpoint workflow is to only 'override' or 'add on' additional configs to the existing endpoint configs. In the case of the transformer, if there's a transformer in the original 'configs' but no transformer specified in the new 'configs', the existing transformer will simply [not get overwritten](https://github.com/caraml-dev/merlin/blob/072325c6e1ae1226216db7186757b22a391f6c72/api/service/version_endpoint_service.go#L200) and remain in the newly deployed endpoint. 

Actually this behaviour exists for all the other endpoint configs, even the 'null-able' ones, like env vars and the logger. I suppose this is a design choice for the update endpoint workflow and not a bug? In order to remove the transformer, env vars or the logger, I think user has to create an entirely new model version right? 🤔

Reply:
Oh, I see. I have forgotten this. Thanks for checking it.

-> I suppose this is a design choice for the update endpoint workflow and not a bug?
Yes, this is a design choice.

-> In order to remove the transformer, env vars or the logger, I think the user has to create an entirely new model version right?
It's not necessarily to create an entirely new model version. You can still using redeploy but with empty or disabled value. For example:

```
merlin.deploy(v, transformer=StandardTransformer(enabled=False))
```